parisienne/README.md

```diff
@@ -0,0 +1,38 @@
+# Parisienne
+
+![A screenshot of the Parisienne theme.](https://user-images.githubusercontent.com/5375500/71043725-88134380-20fd-11ea-996f-44b6a1640856.png "Parisienne Screenshot")
+
+## About
+
+This theme an experiment using block-based templates and theme starter content to bootstrap a WordPress site. 
+
+[Demo](https://injured-anaconda.jurassic.ninja/)
+
+It is based on [Johannes](https://github.com/melchoyce/johannes), a block-based theme by [Mel Choyce](https://melchoyce.design/). All images used in this theme were sourced from [this are.na board](https://www.are.na/stephanie-zabala/i-find-these-elegant).
```

Comment:
In case this gets packaged up and distributed somewhere, we may want to just make sure these are all CC0 images.